src/groups/mqb/mqba/mqba_clientsession.cpp

```diff
@@ -1289,8 +1289,8 @@ void ClientSession::openQueueCb(
         response.choice().makeStatus(status);
 
         BALL_LOG_WARN << "#CLIENT_OPENQUEUE_FAILURE " << description()
-                      << ": Error while opening queue: [reason: '" << status
-                      << "', request: " << handleParamsCtrlMsg << "]";
+                      << ": Error while opening queue: [reason: " << status
```

Comment:
the quote was there for a good reason -  `status` is a sentence

Reply:
https://github.com/bloomberg/blazingmq/blob/21feef613674bbafe68f34ee5c849b837676a6bc/src/groups/mqb/mqba/mqba_clientsession.cpp#L1269

Here, `const bmqp_ctrlmsg::Status& status` is a schema object, so it's not a string, it's a structure with some fields.

An example output _before the PR_ is here:
`
WARN mqba_clientsession.cpp:1294 #CLIENT_OPENQUEUE_FAILURE testapp.tsk:89302: Error while opening queue: [reason: '[ category = E_UNKNOWN code = -1 message = "AppId should not be specified when opening a non-fanout queue" ]', request: [ rId = 2 choice = [ openQueue = [ handleParameters = [ uri = "bmq://test.mmap.priority?id=abd" qId = 0 subIdInfo = [ subId = 1 appId = "abd" ] flags = 2 readCount = 1 writeCount = 0 adminCount = 0 ] ] ] ]]
`

Reply:
Got it. Thanks